### DISCUSSION TITLE
S3 Bucket SSE Disabled Question

### DISCUSSION THREAD

**Discussion by 0xVak** [Checkmarx/kics#7081] [General] [answered]
I'm having a hard time grasping what the Terraform AWS query "S3 Bucket SSE Disabled" is meant for. Given that all S3 buckets by default are encrypted with SSE-S3(AES256), is this just checking for the related Terraform resource and fails if none are associated? Because technically it is, just not defined via TF

**Comment by gabriel-cx** [comment, marked as answer]
Hi @0xVak ,

Thank you for your inputs!
Kindly notice that those queries related to `S3 Bucket SSE Disabled` [were removed from KICS, since v2.0.0](https://github.com/Checkmarx/kics/pull/6932).

Hope this clarifies your doubts.
